limits.c

```diff
@@ -77,6 +77,7 @@ void limits_enable(uint8_t axes, uint8_t expected) {
   limits.expected = bit_istrue(settings.flags,BITFLAG_INVERT_LIMIT_PINS)?~expected:expected;
   limits.active = axes<<LIMIT_BIT_SHIFT;
   limits.mag_gap_check = MAG_GAP_CHECK_ENABLE;
+  memcpy(sys.probe_position, sys.position, sizeof(float) * N_AXIS);
```

Comment:
When we activate limits, sys.probe_position and sys.position may not be close to each other (i.e. the difference might be greater than the threshold). This causes an alarm to be activated immediately after we call limits_enable.

We therefore need to reset the probe position to be the same as the current position when we enable limits.